docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,626 @@ license: |
+    </tr> 
+    <tr>
+      <td> <b>covar_pop</b>(<i>e1: Column, e2: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the population covariance of a set of number pairs</td>
+    </tr> 
+    <tr>
+      <td> <b>covar_samp</b>(<i>e1: Column, e2: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the sample covariance of a set of number pairs</td>
+    </tr>  
+    <tr>
+      <td> <b>{first | first_value}</b>(<i>e: Column[, isIgnoreNull]</i>)</td>
+      <td>Column name[, True/False(default)]</td>
+      <td>Returns the first value of column for a group of rows. If `isIgnoreNull` is true, returns only non-null values, default is false. This function is non-deterministic</td>
+    </tr>      
+    <tr>
+       <td> <b>skewness</b>(<i>e: Column</i>)</td>
+       <td>Column name</td>
+       <td>Returns the skewness value calculated from values of a group</td>
+    </tr>    
+    <tr>
+       <td> <b>kurtosis</b>(<i>e: Column</i>)</td>
+       <td>Column name</td>
+       <td>Returns the kurtosis value calculated from values of a group</td>
+    </tr>    
+    <tr>
+      <td> <b>{last | last_value}</b>(<i>e: Column[, isIgnoreNull]</i>)</td>
+      <td>Column name[, True/False(default)]</td>
+      <td>Returns the last value of column for a group of rows. If `isIgnoreNull` is true, returns only non-null values, default is false. This function is non-deterministic</td>
+    </tr>      
+    <tr>
+      <td> <b>max</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the maximum value of the column.</td>
+    </tr>          
+    <tr>
+      <td> <b>max_by</b>(<i>e1: Column, e2: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the value of column e1 associated with the maximum value of column e2.</td>
+    </tr>   
+    <tr>
+      <td> <b>min</b>(<i>e: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the minimum value of the column.</td>
+    </tr>          
+    <tr>
+      <td> <b>min_by</b>(<i>e1: Column, e2: Column</i>)</td>
+      <td>Column name</td>
+      <td>Returns the value of column e1 associated with the minimum value of column e2.</td>
+    </tr>      
+    <tr>
+      <td> <b>percentile</b>(<i>e: Column, percentage [, frequency]</i>)</td>
+      <td>Column name; percentage is a number between 0 and 1; frequency is a positive integer</td>
+      <td>Returns the exact percentile value of numeric column at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td> <b>percentile</b>(<i>e: Column, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>Column name; percentage array is an array of number between 0 and 1; frequency is a positive integer</td>
+      <td>Returns the exact percentile value array of numeric column at the given percentage(s).</td>
+    </tr>        
+    <tr>
+      <td> <b>{percentile_approx | percentile_approx}</b>(<i>e: Column, percentage [, frequency]</i>)</td>
```

Comment:
Is this a 3.1 function?
```
   * @group agg_funcs
   * @since 3.1.0
   */
  def percentile_approx(e: Column, percentage: Column, accuracy: Column): Column = {
```